universum/poll.py

```diff
@@ -65,6 +65,8 @@ def execute(self):
             self.latest_cls = self.vcs.driver.get_changes(self.stored_cls, self.settings.max_number)
             for depot in self.latest_cls.keys():
                 self.structure.run_in_block(self.process_single_mapping, "Processing depot " + depot, True, depot)
+        except NotImplementedError:
+            self.out.log("Polling is skipped because current VCS doesn't support it")
```

Comment:
isn't this Warning worthy? should we use log_stderr here?

Reply:
It is similar to the message with diff calculation for local vcs. IMO it is better to show the message than to silently do nothing.

Reply:
I see